### ISSUE TITLE
selectOneRadio: custom layout stops functioning after post

### ISSUE THREAD

**Issue by leomekenkamp** [primefaces/primefaces#3500]
## 1) Environment
- PrimeFaces version: 6.2.1
- Does it work on the newest released PrimeFaces version? No
- Does it work on the newest sources in GitHub? (Build by source -> https://github.com/primefaces/primefaces/wiki/Building-From-Source) No
- Application server + version: Tomcat 8.5.15
- Affected browsers: Firefox 59, IE 11, Chrome 64.0.3282.186

## 2) Expected behavior
After a full form post, the custom layout radio buttons should be selectable.

## 3) Actual behavior
After a full form post, the custom layout radio buttons do not respond to mouse clicks.

## 4) Steps to reproduce
- verify radio buttons respond to mouse clicks
- click the submit button
- verify that the submitted value is not set and that the radio buttons do not respond to mouse clicks

## 5) Sample XHTML
(code formatted after tip from Rapster)

```
<html
        xmlns="http://www.w3.org/1999/xhtml"
        xmlns:h="http://xmlns.jcp.org/jsf/html"
        xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
        xmlns:f="http://xmlns.jcp.org/jsf/core"
        xmlns:p="http://primefaces.org/ui"
>
    <h:head>
    </h:head>
    <h:body>
        <h:form>
            <p:selectOneRadio layout="custom" id="selectRadio">
                <f:selectItem itemValue="1" itemLabel="one"/>
                <f:selectItem itemValue="2" itemLabel="two"/>
            </p:selectOneRadio>
            <p:radioButton for="selectRadio" itemIndex="0" />
            <p:radioButton for="selectRadio" itemIndex="1"/>
            <p:commandButton value="submit" ajax="false"/>
        </h:form>
    </h:body>
</html>
```

## 6) Sample bean
// No backing bean needed.

**Comment by leomekenkamp** [comment]
Where did my xhtml sample go? Trying again:

<html
        xmlns="http://www.w3.org/1999/xhtml"
        xmlns:h="http://xmlns.jcp.org/jsf/html"
        xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
        xmlns:f="http://xmlns.jcp.org/jsf/core"
        xmlns:p="http://primefaces.org/ui"
>
    <h:head>
    </h:head>
    <h:body>
        <h:form>
            <p:selectOneRadio layout="custom" id="selectRadio">
                <f:selectItem itemValue="1" itemLabel="one"/>
                <f:selectItem itemValue="2" itemLabel="two"/>
            </p:selectOneRadio>
            <p:radioButton for="selectRadio" itemIndex="0" />
            <p:radioButton for="selectRadio" itemIndex="1"/>
            <p:commandButton value="submit" ajax="false"/>
        </h:form>
    </h:body>
</html>

**Comment by Rapster** [comment]
@leomekenkamp FYI, use ` for inline code, ``` for code blocks. See https://help.github.com/articles/creating-and-highlighting-code-blocks/

**Comment by leomekenkamp** [comment]
Ah, thanks. Fixed it. The layout that is :-), the original bug still stands.

**Comment by tandraschko** [comment]
Should be fixed by: https://github.com/primefaces/primefaces/issues/3530
Could you build by source and test?
Mert will add it to 6.2.2

**Comment by melloware** [comment]
@leomekenkamp did you get a chance to test this?  I think this is a critical fix for PF 6.2.2 Elite.

**Comment by tandraschko** [comment]
Will close it for now, it should be fixed. Please reopen if you still have problems with the newer versions soon.

**Comment by leomekenkamp** [comment]
Sorry, too busy with other stuff. Will get back to you when I have tested.

**Comment by leomekenkamp** [comment]
Finally have had time to verify: confirmed fully fixed in 6.2.2. Thanks!
